[ITP Cleanup] Rename some resource load statistics feature flags to refer to "tracking prevention"

whsieh opened the pull request:
#### 30e20b03def3058764d0b1ee5431fbe1231ed2b7
<pre>
[ITP Cleanup] Rename some resource load statistics feature flags to refer to &quot;tracking prevention&quot;
<a href="https://bugs.webkit.org/show_bug.cgi?id=246906">https://bugs.webkit.org/show_bug.cgi?id=246906</a>

Reviewed by Megan Gardner.

Currently, the feature flag that determines whether tracking prevention measures in WebKit are
enabled or disabled refers to the feature as &quot;resource load statistics&quot;. We rename these symbols in
WebKit internal code from `ResourceLoadStatisticsEnabled` (and similar) to
`TrackingPreventionEnabled` to reflect this real intent.

We also rename some places that refer to `ITPState` and `ShouldAskITP` to `TrackingPreventionState`
and `ApplyTrackingPrevention`, respectively.

* LayoutTests/http/tests/media/media-stream/enumerate-devices-source-id.html:
* LayoutTests/http/tests/navigation/statistics.html:
* LayoutTests/http/tests/referrer-policy-anchor/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/same-origin.html:
* LayoutTests/http/tests/referrer-policy-script/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/same-origin.html:
* LayoutTests/http/tests/resourceLoadStatistics/capped-lifetime-for-cookie-set-in-js-24-hours.html:
* LayoutTests/http/tests/resourceLoadStatistics/capped-lifetime-for-cookie-set-in-js.html:
* LayoutTests/http/tests/resourceLoadStatistics/do-not-block-top-level-navigation-redirect.html:
* LayoutTests/http/tests/resourceLoadStatistics/prevalent-resource-without-user-interaction.html:
* LayoutTests/http/tests/resourceLoadStatistics/resources/util.js:
(setEnableFeature):
* LayoutTests/http/tests/webAPIStatistics/canvas-read-and-write-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/font-load-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/navigator-functions-accessed-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/screen-functions-accessed-data-collection.html:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::referrer):
* Source/WebCore/loader/CookieJar.cpp:
(WebCore::CookieJar::cookies const):
(WebCore::CookieJar::setCookies):
(WebCore::CookieJar::cookieRequestHeaderFieldValue const):
(WebCore::CookieJar::getRawCookies const):
* Source/WebCore/page/DeprecatedGlobalSettings.cpp:
(WebCore::DeprecatedGlobalSettings::DeprecatedGlobalSettings):
(WebCore::DeprecatedGlobalSettings::setTrackingPreventionEnabled):
(WebCore::DeprecatedGlobalSettings::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/page/DeprecatedGlobalSettings.h:
(WebCore::DeprecatedGlobalSettings::trackingPreventionEnabled):
(WebCore::DeprecatedGlobalSettings::resourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::triggerOptionalStorageAccessQuirk const):
* Source/WebCore/platform/network/CacheValidation.cpp:
(WebCore::cookieRequestHeaderFieldValue):
* Source/WebCore/platform/network/NetworkStorageSession.cpp:
(WebCore::NetworkStorageSession::setTrackingPreventionEnabled):
(WebCore::NetworkStorageSession::trackingPreventionEnabled const):
(WebCore::NetworkStorageSession::setTrackingPreventionDebugLoggingEnabled):
(WebCore::NetworkStorageSession::trackingPreventionDebugLoggingEnabled const):
(WebCore::NetworkStorageSession::shouldBlockThirdPartyCookies const):
(WebCore::NetworkStorageSession::shouldBlockThirdPartyCookiesButKeepFirstPartyCookiesFor const):
(WebCore::NetworkStorageSession::shouldBlockCookies const):
(WebCore::NetworkStorageSession::setResourceLoadStatisticsEnabled): Deleted.
(WebCore::NetworkStorageSession::resourceLoadStatisticsEnabled const): Deleted.
(WebCore::NetworkStorageSession::setResourceLoadStatisticsDebugLoggingEnabled): Deleted.
(WebCore::NetworkStorageSession::resourceLoadStatisticsDebugLoggingEnabled const): Deleted.
* Source/WebCore/platform/network/NetworkStorageSession.h:
* Source/WebCore/platform/network/cf/NetworkStorageSessionCFNetWin.cpp:
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::getRawCookies const):
* Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm:
(WebCore::NetworkStorageSession::cookiesForURL const):
(WebCore::NetworkStorageSession::cookiesForSession const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
* Source/WebCore/platform/network/curl/NetworkStorageSessionCurl.cpp:
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
* Source/WebCore/platform/network/curl/ResourceHandleCurl.cpp:
(WebCore::ResourceHandle::createCurlRequest):
* Source/WebCore/platform/network/soup/NetworkStorageSessionSoup.cpp:
(WebCore::NetworkStorageSession::setCookieAcceptPolicy):
(WebCore::NetworkStorageSession::setTrackingPreventionEnabled):
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
(WebCore::cookiesForSession):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::setTrackingPreventionEnabled):
(WebCore::Internals::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:
* Source/WebKit/NetworkProcess/Classifier/WebResourceLoadStatisticsStore.cpp:
(WebKit::WebResourceLoadStatisticsStore::setResourceLoadStatisticsDebugMode):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp:
(WebKit::NetworkConnectionToWebProcess::cookiesForDOM):
(WebKit::NetworkConnectionToWebProcess::setCookiesFromDOM):
(WebKit::NetworkConnectionToWebProcess::cookieRequestHeaderFieldValue):
(WebKit::NetworkConnectionToWebProcess::getRawCookies):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h:
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkDataTask.cpp:
(WebKit::NetworkDataTask::restrictRequestReferrerToOriginIfNeeded):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::setTrackingPreventionEnabled):
(WebKit::NetworkProcess::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::logCookieInformationInternal):
* Source/WebKit/NetworkProcess/NetworkSession.cpp:
(WebKit::m_shouldAcceptInsecureCertificatesForWebSockets):
(WebKit::NetworkSession::setTrackingPreventionEnabled):
(WebKit::NetworkSession::isTrackingPreventionEnabled const):
(WebKit::NetworkSession::setResourceLoadStatisticsEnabled): Deleted.
(WebKit::NetworkSession::isResourceLoadStatisticsEnabled const): Deleted.
* Source/WebKit/NetworkProcess/NetworkSession.h:
* Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm:
(WebKit::NetworkDataTaskCocoa::shouldApplyCookiePolicyForThirdPartyCloaking const):
(WebKit::NetworkDataTaskCocoa::applyCookiePolicyForThirdPartyCloaking):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(WebKit::activateSessionCleanup):
* Source/WebKit/NetworkProcess/curl/NetworkDataTaskCurl.cpp:
(WebKit::NetworkDataTaskCurl::appendCookieHeader):
* Source/WebKit/NetworkProcess/curl/NetworkSessionCurl.cpp:
(WebKit::NetworkSessionCurl::NetworkSessionCurl):
* Source/WebKit/NetworkProcess/curl/WebSocketTaskCurl.cpp:
(WebKit::WebSocketTask::didOpen):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.h:
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm:
(WebKit::determineTrackingPreventionStateInternal):
(WebKit::determineTrackingPreventionState):
(WebKit::doesAppHaveTrackingPreventionEnabled):
(WebKit::doesParentProcessHaveTrackingPreventionEnabled):
(WebKit::determineITPStateInternal): Deleted.
(WebKit::determineITPState): Deleted.
(WebKit::doesAppHaveITPEnabled): Deleted.
(WebKit::doesParentProcessHaveITPEnabled): Deleted.
* Source/WebKit/Shared/ResourceLoadStatisticsParameters.h:
(WebKit::ResourceLoadStatisticsParameters::encode const):
(WebKit::ResourceLoadStatisticsParameters::decode):
* Source/WebKit/Shared/WebProcessDataStoreParameters.h:
(WebKit::WebProcessDataStoreParameters::encode const):
(WebKit::WebProcessDataStoreParameters::decode):
* Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreRef.cpp:
(WKWebsiteDataStoreSetResourceLoadStatisticsEnabled):
(WKWebsiteDataStoreGetResourceLoadStatisticsEnabled):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore _resourceLoadStatisticsEnabled]):
(-[WKWebsiteDataStore _setResourceLoadStatisticsEnabled:]):
* Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp:
(webkit_website_data_manager_set_itp_enabled):
(webkit_website_data_manager_get_itp_enabled):
* Source/WebKit/UIProcess/WebProcessPool.cpp:
(WebKit::WebProcessPool::createNewWebProcess):
(WebKit::WebProcessPool::webProcessDataStoreParameters):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::WebsiteDataStore):
(WebKit::WebsiteDataStore::setIsRunningResourceLoadStatisticsTest):
(WebKit::WebsiteDataStore::trackingPreventionEnabled const):
(WebKit::WebsiteDataStore::resourceLoadStatisticsDebugMode const):
(WebKit::WebsiteDataStore::setTrackingPreventionEnabled):
(WebKit::WebsiteDataStore::setResourceLoadStatisticsDebugMode):
(WebKit::WebsiteDataStore::isResourceLoadStatisticsEphemeral const):
(WebKit::WebsiteDataStore::clearResourceLoadStatisticsInWebProcesses):
(WebKit::WebsiteDataStore::parameters):
(WebKit::WebsiteDataStore::resourceLoadStatisticsEnabled const): Deleted.
(WebKit::WebsiteDataStore::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
(WebKit::WebsiteDataStore::isTrackingPreventionStateExplicitlySet const):
(WebKit::WebsiteDataStore::useExplicitTrackingPreventionState):
(WebKit::WebsiteDataStore::isItpStateExplicitlySet const): Deleted.
(WebKit::WebsiteDataStore::useExplicitITPState): Deleted.
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.cpp:
(WebKit::WebsiteDataStoreConfiguration::copy const):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(WebKit::WebsiteDataStoreConfiguration::resourceLoadStatisticsDebugModeEnabled const):
(WebKit::WebsiteDataStoreConfiguration::setResourceLoadStatisticsDebugModeEnabled):
* Source/WebKit/WebProcess/WebPage/WebCookieCache.cpp:
(WebKit::WebCookieCache::cookiesForDOM):
(WebKit::WebCookieCache::setCookiesFromDOM):
* Source/WebKit/WebProcess/WebPage/WebCookieJar.cpp:
(WebKit::shouldBlockCookies):
(WebKit::WebCookieJar::cookies const):
(WebKit::WebCookieJar::setCookies):
(WebKit::WebCookieJar::cookiesEnabled const):
(WebKit::WebCookieJar::cookieRequestHeaderFieldValue const):
(WebKit::WebCookieJar::getRawCookies const):
* Source/WebKit/WebProcess/WebProcess.cpp:
(WebKit::WebProcess::setWebsiteDataStoreParameters):
(WebKit::WebProcess::setTrackingPreventionEnabled):
(WebKit::WebProcess::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/WebProcess.messages.in:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
(-[WebView _preferencesChanged:]):

Canonical link: <a href="https://commits.webkit.org/256110@main">https://commits.webkit.org/256110@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/545517ac7da1f846c4ee995b0133302beee90d78

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/94718 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/3874 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/27595 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/104337 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/164601 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/3942 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/32063 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/87030 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/100271 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/100383 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/2839 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/81078 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/29854 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/84768 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/84318 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/72743 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/38468 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/18133 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/36314 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/70/builds/19412 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4213 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/40225 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/42056 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/42197 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/62/builds/38643 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebKit/NetworkProcess/NetworkDataTask.cpp:
```diff
@@ -180,7 +180,7 @@ NetworkSession* NetworkDataTask::networkSession()
 {
 #if ENABLE(TRACKING_PREVENTION)
-    if ((m_session->sessionID().isEphemeral() || m_session->isResourceLoadStatisticsEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
+    if ((m_session->sessionID().isEphemeral() || m_session->istrackingPreventionEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
```
```suggestion
    if ((m_session->sessionID().isEphemeral() || m_session->isTrackingPreventionEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
```

camelCase is not respected here.

whsieh replied:
Fixed all of these (and the libsoup build as well).

nt1m commented on Source/WebKit/NetworkProcess/NetworkSession.cpp:
```diff
@@ -263,7 +263,7 @@ void NetworkSession::forwardResourceLoadStatisticsSettings()
 }
 
-bool NetworkSession::isResourceLoadStatisticsEnabled() const
+bool NetworkSession::istrackingPreventionEnabled() const
```
ditto here

nt1m commented on Source/WebKit/NetworkProcess/NetworkSession.h:
```diff
@@ -121,8 +121,8 @@ class NetworkSession : public CanMakeWeakPtr<NetworkSession> {
-    void setResourceLoadStatisticsEnabled(bool);
-    bool isResourceLoadStatisticsEnabled() const;
+    void setTrackingPreventionEnabled(bool);
+    bool istrackingPreventionEnabled() const;
```
ditto

nt1m commented on Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
```diff
@@ -1205,17 +1205,17 @@ static void activateSessionCleanup(NetworkSessionCocoa& session, const NetworkSe
-    bool itpEnabled = doesParentProcessHaveITPEnabled(session.networkProcess(), parameters.appHasRequestedCrossWebsiteTrackingPermission);
-    bool passedEnabledState = session.isResourceLoadStatisticsEnabled();
+    bool trackingPreventionEnabled = doesParentProcessHaveTrackingPreventionEnabled(session.networkProcess(), parameters.appHasRequestedCrossWebsiteTrackingPermission);
+    bool passedEnabledState = session.istrackingPreventionEnabled();
```
ditto here

whsieh committed:
[ITP Cleanup] Rename some resource load statistics feature flags to refer to "tracking prevention"
https://bugs.webkit.org/show_bug.cgi?id=246906

Reviewed by Megan Gardner.

Currently, the feature flag that determines whether tracking prevention measures in WebKit are
enabled or disabled refers to the feature as "resource load statistics". We rename these symbols in
WebKit internal code from `ResourceLoadStatisticsEnabled` (and similar) to
`TrackingPreventionEnabled` to reflect this real intent.

We also rename some places that refer to `ITPState` and `ShouldAskITP` to `TrackingPreventionState`
and `ApplyTrackingPrevention`, respectively.

* LayoutTests/http/tests/media/media-stream/enumerate-devices-source-id.html:
* LayoutTests/http/tests/navigation/statistics.html:
* LayoutTests/http/tests/referrer-policy-anchor/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-iframe/unsafe-url/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy-img/unsafe-url/same-origin.html:
* LayoutTests/http/tests/referrer-policy-script/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/no-referrer-when-downgrade/same-origin.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/no-referrer/same-origin.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/same-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/same-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/same-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/strict-origin-when-cross-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/strict-origin/same-origin.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/cross-origin-http-http.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/cross-origin-http.https.html:
* LayoutTests/http/tests/referrer-policy/unsafe-url/same-origin.html:
* LayoutTests/http/tests/resourceLoadStatistics/capped-lifetime-for-cookie-set-in-js-24-hours.html:
* LayoutTests/http/tests/resourceLoadStatistics/capped-lifetime-for-cookie-set-in-js.html:
* LayoutTests/http/tests/resourceLoadStatistics/do-not-block-top-level-navigation-redirect.html:
* LayoutTests/http/tests/resourceLoadStatistics/prevalent-resource-without-user-interaction.html:
* LayoutTests/http/tests/resourceLoadStatistics/resources/util.js:
(setEnableFeature):
* LayoutTests/http/tests/webAPIStatistics/canvas-read-and-write-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/font-load-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/navigator-functions-accessed-data-collection.html:
* LayoutTests/http/tests/webAPIStatistics/screen-functions-accessed-data-collection.html:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::referrer):
* Source/WebCore/loader/CookieJar.cpp:
(WebCore::CookieJar::cookies const):
(WebCore::CookieJar::setCookies):
(WebCore::CookieJar::cookieRequestHeaderFieldValue const):
(WebCore::CookieJar::getRawCookies const):
* Source/WebCore/page/DeprecatedGlobalSettings.cpp:
(WebCore::DeprecatedGlobalSettings::DeprecatedGlobalSettings):
(WebCore::DeprecatedGlobalSettings::setTrackingPreventionEnabled):
(WebCore::DeprecatedGlobalSettings::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/page/DeprecatedGlobalSettings.h:
(WebCore::DeprecatedGlobalSettings::trackingPreventionEnabled):
(WebCore::DeprecatedGlobalSettings::resourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::triggerOptionalStorageAccessQuirk const):
* Source/WebCore/platform/network/CacheValidation.cpp:
(WebCore::cookieRequestHeaderFieldValue):
* Source/WebCore/platform/network/NetworkStorageSession.cpp:
(WebCore::NetworkStorageSession::setTrackingPreventionEnabled):
(WebCore::NetworkStorageSession::trackingPreventionEnabled const):
(WebCore::NetworkStorageSession::setTrackingPreventionDebugLoggingEnabled):
(WebCore::NetworkStorageSession::trackingPreventionDebugLoggingEnabled const):
(WebCore::NetworkStorageSession::shouldBlockThirdPartyCookies const):
(WebCore::NetworkStorageSession::shouldBlockThirdPartyCookiesButKeepFirstPartyCookiesFor const):
(WebCore::NetworkStorageSession::shouldBlockCookies const):
(WebCore::NetworkStorageSession::setResourceLoadStatisticsEnabled): Deleted.
(WebCore::NetworkStorageSession::resourceLoadStatisticsEnabled const): Deleted.
(WebCore::NetworkStorageSession::setResourceLoadStatisticsDebugLoggingEnabled): Deleted.
(WebCore::NetworkStorageSession::resourceLoadStatisticsDebugLoggingEnabled const): Deleted.
* Source/WebCore/platform/network/NetworkStorageSession.h:
* Source/WebCore/platform/network/cf/NetworkStorageSessionCFNetWin.cpp:
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::getRawCookies const):
* Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm:
(WebCore::NetworkStorageSession::cookiesForURL const):
(WebCore::NetworkStorageSession::cookiesForSession const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
* Source/WebCore/platform/network/curl/NetworkStorageSessionCurl.cpp:
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
* Source/WebCore/platform/network/curl/ResourceHandleCurl.cpp:
(WebCore::ResourceHandle::createCurlRequest):
* Source/WebCore/platform/network/soup/NetworkStorageSessionSoup.cpp:
(WebCore::NetworkStorageSession::setCookieAcceptPolicy):
(WebCore::NetworkStorageSession::setTrackingPreventionEnabled):
(WebCore::NetworkStorageSession::setCookiesFromDOM const):
(WebCore::NetworkStorageSession::getRawCookies const):
(WebCore::cookiesForSession):
(WebCore::NetworkStorageSession::cookiesForDOM const):
(WebCore::NetworkStorageSession::cookieRequestHeaderFieldValue const):
(WebCore::NetworkStorageSession::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::setTrackingPreventionEnabled):
(WebCore::Internals::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:
* Source/WebKit/NetworkProcess/Classifier/WebResourceLoadStatisticsStore.cpp:
(WebKit::WebResourceLoadStatisticsStore::setResourceLoadStatisticsDebugMode):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp:
(WebKit::NetworkConnectionToWebProcess::cookiesForDOM):
(WebKit::NetworkConnectionToWebProcess::setCookiesFromDOM):
(WebKit::NetworkConnectionToWebProcess::cookieRequestHeaderFieldValue):
(WebKit::NetworkConnectionToWebProcess::getRawCookies):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h:
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkDataTask.cpp:
(WebKit::NetworkDataTask::restrictRequestReferrerToOriginIfNeeded):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::setTrackingPreventionEnabled):
(WebKit::NetworkProcess::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::logCookieInformationInternal):
* Source/WebKit/NetworkProcess/NetworkSession.cpp:
(WebKit::m_shouldAcceptInsecureCertificatesForWebSockets):
(WebKit::NetworkSession::setTrackingPreventionEnabled):
(WebKit::NetworkSession::isTrackingPreventionEnabled const):
(WebKit::NetworkSession::setResourceLoadStatisticsEnabled): Deleted.
(WebKit::NetworkSession::isResourceLoadStatisticsEnabled const): Deleted.
* Source/WebKit/NetworkProcess/NetworkSession.h:
* Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm:
(WebKit::NetworkDataTaskCocoa::shouldApplyCookiePolicyForThirdPartyCloaking const):
(WebKit::NetworkDataTaskCocoa::applyCookiePolicyForThirdPartyCloaking):
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
(WebKit::activateSessionCleanup):
* Source/WebKit/NetworkProcess/curl/NetworkDataTaskCurl.cpp:
(WebKit::NetworkDataTaskCurl::appendCookieHeader):
* Source/WebKit/NetworkProcess/curl/NetworkSessionCurl.cpp:
(WebKit::NetworkSessionCurl::NetworkSessionCurl):
* Source/WebKit/NetworkProcess/curl/WebSocketTaskCurl.cpp:
(WebKit::WebSocketTask::didOpen):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.h:
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm:
(WebKit::determineTrackingPreventionStateInternal):
(WebKit::determineTrackingPreventionState):
(WebKit::doesAppHaveTrackingPreventionEnabled):
(WebKit::doesParentProcessHaveTrackingPreventionEnabled):
(WebKit::determineITPStateInternal): Deleted.
(WebKit::determineITPState): Deleted.
(WebKit::doesAppHaveITPEnabled): Deleted.
(WebKit::doesParentProcessHaveITPEnabled): Deleted.
* Source/WebKit/Shared/ResourceLoadStatisticsParameters.h:
(WebKit::ResourceLoadStatisticsParameters::encode const):
(WebKit::ResourceLoadStatisticsParameters::decode):
* Source/WebKit/Shared/WebProcessDataStoreParameters.h:
(WebKit::WebProcessDataStoreParameters::encode const):
(WebKit::WebProcessDataStoreParameters::decode):
* Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreRef.cpp:
(WKWebsiteDataStoreSetResourceLoadStatisticsEnabled):
(WKWebsiteDataStoreGetResourceLoadStatisticsEnabled):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore _resourceLoadStatisticsEnabled]):
(-[WKWebsiteDataStore _setResourceLoadStatisticsEnabled:]):
* Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp:
(webkit_website_data_manager_set_itp_enabled):
(webkit_website_data_manager_get_itp_enabled):
* Source/WebKit/UIProcess/WebProcessPool.cpp:
(WebKit::WebProcessPool::createNewWebProcess):
(WebKit::WebProcessPool::webProcessDataStoreParameters):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::WebsiteDataStore):
(WebKit::WebsiteDataStore::setIsRunningResourceLoadStatisticsTest):
(WebKit::WebsiteDataStore::trackingPreventionEnabled const):
(WebKit::WebsiteDataStore::resourceLoadStatisticsDebugMode const):
(WebKit::WebsiteDataStore::setTrackingPreventionEnabled):
(WebKit::WebsiteDataStore::setResourceLoadStatisticsDebugMode):
(WebKit::WebsiteDataStore::isResourceLoadStatisticsEphemeral const):
(WebKit::WebsiteDataStore::clearResourceLoadStatisticsInWebProcesses):
(WebKit::WebsiteDataStore::parameters):
(WebKit::WebsiteDataStore::resourceLoadStatisticsEnabled const): Deleted.
(WebKit::WebsiteDataStore::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
(WebKit::WebsiteDataStore::isTrackingPreventionStateExplicitlySet const):
(WebKit::WebsiteDataStore::useExplicitTrackingPreventionState):
(WebKit::WebsiteDataStore::isItpStateExplicitlySet const): Deleted.
(WebKit::WebsiteDataStore::useExplicitITPState): Deleted.
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.cpp:
(WebKit::WebsiteDataStoreConfiguration::copy const):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(WebKit::WebsiteDataStoreConfiguration::resourceLoadStatisticsDebugModeEnabled const):
(WebKit::WebsiteDataStoreConfiguration::setResourceLoadStatisticsDebugModeEnabled):
* Source/WebKit/WebProcess/WebPage/WebCookieCache.cpp:
(WebKit::WebCookieCache::cookiesForDOM):
(WebKit::WebCookieCache::setCookiesFromDOM):
* Source/WebKit/WebProcess/WebPage/WebCookieJar.cpp:
(WebKit::shouldBlockCookies):
(WebKit::WebCookieJar::cookies const):
(WebKit::WebCookieJar::setCookies):
(WebKit::WebCookieJar::cookiesEnabled const):
(WebKit::WebCookieJar::cookieRequestHeaderFieldValue const):
(WebKit::WebCookieJar::getRawCookies const):
* Source/WebKit/WebProcess/WebProcess.cpp:
(WebKit::WebProcess::setWebsiteDataStoreParameters):
(WebKit::WebProcess::setTrackingPreventionEnabled):
(WebKit::WebProcess::setResourceLoadStatisticsEnabled): Deleted.
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/WebProcess.messages.in:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
(-[WebView _preferencesChanged:]):

Canonical link: https://commits.webkit.org/256110@main
